### PR TITLE
[travis] Added tests for django 2.0

### PR DESCRIPTION
**Pull request by AYZhu**

### CODE REVIEW

**Comment by AYZhu** [comment]
the `.encode().decode()` is necessary to get the string in the right data type

File: owm_legacy/tests.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from time import sleep
 
-from django.core.urlresolvers import reverse
+from django.urls import reverse
```
Comment (auvipy) [inline owm_legacy/tests.py:3]: does this import work in less then 2?

Comment (AYZhu) [reply owm_legacy/tests.py:3]: Yes, it works in django >= 1.9

Comment (AYZhu) [reply owm_legacy/tests.py:3]: @auvipy

**Review by nemesifier — APPROVED** [review]
Good! I will leave here until django-netjsonconfig is released so I can remove the temporary install of the master branch

**Review by auvipy — APPROVED** [review]
cool

**Pull request merged**